counterpartylib/lib/messages/send.py

```diff
@@ -30,13 +32,29 @@ def initialise (db):
                       asset_idx ON sends (asset)
                    ''')
 
+    # Adds a memo to sends
+    #   SQLite can’t do `ALTER TABLE IF COLUMN NOT EXISTS`.
+    columns = [column['name'] for column in cursor.execute('''PRAGMA table_info(sends)''')]
+    if 'memo' not in columns:
+        cursor.execute('''ALTER TABLE sends ADD COLUMN memo BLOB''')
+
+    cursor.execute('''CREATE INDEX IF NOT EXISTS
+                      memo_idx ON sends (memo)
+                   ''')
+
 def unpack(db, message, block_index):
     return send1.unpack(db, message, block_index)
 
 def validate (db, source, destination, asset, quantity, block_index):
     return send1.validate(db, source, destination, asset, quantity, block_index)
 
-def compose (db, source, destination, asset, quantity):
+def compose (db, source, destination, asset, quantity, memo=None, memo_is_hex=False, use_enhanced_send=None):
+    # special case - enhanced_send replaces send by default when it is enabled
+    #   but it can be explicitly disabled with an API parameter
+    if util.enabled('enhanced_sends'):
+        if use_enhanced_send is None or use_enhanced_send == True:
+            return enhanced_send.compose(db, source, destination, asset, quantity, memo, memo_is_hex)
```

Comment:
I think it should raise an exception when `use_enhanced_send=True` before `enhanced_sends` is enabled?